examples/advanced/main.tf

```diff
@@ -0,0 +1,62 @@
+provider "azurerm" {
+  features {}
+  skip_provider_registration = true
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "rg-example-fwp"
+  location = local.location
+}
+
+resource "azurerm_virtual_network" "example" {
+  name                = "vnet-example"
+  address_space       = ["10.0.0.0/16"]
+  location            = local.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_subnet" "example" {
+  name                 = "AzureFirewallSubnet" # Must be exact 'AzureFirewallSubnet'
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefixes     = ["10.0.0.0/16"]
+}
+
+resource "azurerm_public_ip" "example" {
+  name                = "pip-example"
+  location            = local.location
+  resource_group_name = azurerm_resource_group.example.name
+  allocation_method   = "Static"
+  sku                 = "Standard"
+}
```

Comment:
I think this is a left over from when the test included a firewall. Since we are only using the firewall policy from now on I think this can be deleted.